yoti/YotiHelper.php

```diff
@@ -292,15 +292,26 @@ public static function getFlash()
     /**
      * Generate Yoti unique username.
      *
+     * @param ActivityDetails $activityDetails
      * @param string $prefix
-     * @return string
+     *
+     * @return null|string
      */
-    private function generateUsername($prefix = 'YotiUser-')
+    private function generateUsername(ActivityDetails $activityDetails, $prefix = 'Yoti User')
```

Comment:
`Yoti User` could probably be `yoti.user`. Then you can move `strtolower(str_replace(" ", ".", $prefix))` inside the conditional.